differ/storage.go

```diff
@@ -184,6 +190,10 @@ const (
 		 ORDER BY updated_at
 `
 
+	QueryRecordExistsInReadErrors = `
+		SELECT exists(SELECT 1 FROM read_errors WHERE org_id=$1 and cluster=$2 and updated_at=$3);
```

Comment:
Is `updated_at=$3` ok? shouldn't it be `>`?

Reply:
nope, we are going to check for already reported error for given cluster + timestamp.

Reply:
ah, makes sense